lib/apiservers/engine/backends/commit.go

```diff
@@ -60,6 +60,14 @@ func (i *ImageBackend) Commit(name string, config *backend.ContainerCommitConfig
 	op := trace.NewOperation(context.Background(), "Commit: %s", name)
 	defer trace.End(trace.Audit(name, op))
 
+	valid, err := validateStorageQuota()
+	if err != nil {
+		return "", errors.InternalServerError(err.Error())
+	}
+	if !valid {
+		return "", fmt.Errorf("Storage quota exceeds")
```

Comment:
This error message seems a little confusing. I think it might be better if `validateStorageQuota` returned an error in the case that `vchConfig.Cfg.StorageQuota < imageStorageUsage+vmStorageUsage` since that error message could more clearly communicate the problem. (e.g., by including the storage quota, image usage, and VM usage in the error message). By the time we're here, we've lost that detail, so a user encountering this error will have to investigate the cause of the problem.

This change would also reduce duplication between the two places this method is called.

Reply:
Good point. Fixed.